content/library/api/control-flow/switch_page.md

```diff
@@ -0,0 +1,7 @@
+---
+title: st.switch_page
+slug: /library/api-reference/control-flow/st.switch_page
+description: st.switch_page programatically switches the active page.
```

Comment:
nit:

```suggestion
description: st.switch_page programmatically switches the active page.
```

Reply:
Typo in the meta data! I missed that while going through the preview. Nice catch!